cudamapper/src/overlapper_triggered.cu

```diff
@@ -193,26 +192,6 @@ void OverlapperTriggered::get_overlaps(std::vector<Overlap>& fused_overlaps,
     const auto tail_length_for_chain = 3;
     auto n_anchors                   = d_anchors.size();
 
-    // comparison operator - lambda used to compare Anchors in sort
-    auto comp = [] __host__ __device__(const Anchor& i, const Anchor& j) -> bool {
-        return (i.query_read_id_ < j.query_read_id_) ||
-               ((i.query_read_id_ == j.query_read_id_) &&
-                (i.target_read_id_ < j.target_read_id_)) ||
-               ((i.query_read_id_ == j.query_read_id_) &&
-                (i.target_read_id_ == j.target_read_id_) &&
-                (i.query_position_in_read_ < j.query_position_in_read_)) ||
-               ((i.query_read_id_ == j.query_read_id_) &&
-                (i.target_read_id_ == j.target_read_id_) &&
-                (i.query_position_in_read_ == j.query_position_in_read_) &&
-                (i.target_position_in_read_ < j.target_position_in_read_));
-    };
-
-    auto thrust_exec_policy = thrust::cuda::par.on(stream);
-
-    // sort on device
-    // TODO : currently thrust::sort requires O(2N) auxiliary storage, implement the same functionality using O(N) auxiliary storage
-    thrust::sort(thrust_exec_policy, d_anchors.begin(), d_anchors.end(), comp);
-
```

Comment:
Optional: To express that we assume that the anchors are sorted in the following code, I would add an `assert(thrust::is_sorted(d_anchors.begin(), d_anchors.end());` here. Should we add an `<` operator to the interface of the anchor struct in general? That way you would not need to provide a specific comparison function here, but I am not sure if it is useful otherwise.

Reply:
Agree about assertion here. I don't think there is need a for a comparison function in `Anchor` itself

Reply:
done